front/src/store/modules/eating.store.js

```diff
@@ -0,0 +1,48 @@
+import api from '@api';
+import { SERVER_SCHEDULED_PLACE_TO_EAT_PATH } from '@constants';
+
+const { baseUrl } = require('@/config/config.dev.json');
+
+export default {
+  namespaced: true,
+  actions: {
+    getEatingUser ({ commit }) {
+      return api.get('/scheduledPlaceToEat');
+    },
+    getEatingUserByEatingId ({ commit }, payload) {
```

Comment:
getEatingUser - unused 
getEatingUserByEatingId - unused, but need, because if I delete '{ commit}, ', payload be = first object which has the property commit